ichnaea/cache.py

```diff
@@ -8,15 +8,17 @@
 import redis
 from redis.exceptions import RedisError
 
-from ichnaea.config import REDIS_URI
+from ichnaea.conf import settings
 
 
-def configure_redis(cache_url=REDIS_URI, _client=None):
+def configure_redis(cache_url=None, _client=None):
```

Comment:
Because `REDIS_URI` is a setting, we don't want the value at the time this module was loaded codified as an argument default. That'll mess up the tests.